### DISCUSSION TITLE
Unexpected space when adding a Dashed Line in Fabric.js 6.3.0

### DISCUSSION THREAD

**Discussion by yashChapaniOB** [fabricjs/fabric.js#10151] [General]
When adding a solid line, the same spacing issue shows, but this issue resolves with the stokeLineCap property set. Sqare, this is resolved in the solid line, but when this same property is applied to the dashed line, then the dashed line does not work; it shows a solid line, so my issue is which property to add to remove padding from the dashed line shown in the image. I want to remove extra spacing in the dashed line like the solid line.

![image](https://github.com/user-attachments/assets/dbad8c2a-d08f-451b-875f-f620e13449ad)

**Comment by asturur** [comment]
Please add a code snippet, maybe there is a bug.

**Reply by yashChapaniOB** [reply]
I added code in the issue section; here is my issue number

https://github.com/fabricjs/fabric.js/issues/10150

**Comment by asturur** [comment]
Why you open double discussion + issue

**Reply by yashChapaniOB** [reply]
Can you give me suggestion how to solve this issue?
